src/sentry/models/deleteduser.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+from sentry.db.models import (
+    sane_repr
+)
+from django.utils import timezone
+from sentry.models.deletedentry import DeletedEntry
+
+
+class DeletedUser(DeletedEntry):
+    username = models.CharField(_('username'), max_length=128, unique=True)
```

Comment:
Same comment about the slug being UNIQUE, we shouldn't UNIQUE here either. But also, this field is deprecated in favor of email..

@dcramer did you say that we shouldn't log the email addresses either of deleted users? If not, do you have thoughts about logging a censored version? Like, `matt@sentry.io` -> `m**@sentry.io` ? That way it preserves _some_ identification without storing full email. Otherwise, not sure what information we're looking to get out of this table.